src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx

```diff
@@ -931,12 +931,27 @@ Prior versions of Npgsql may also be instrumented, but duplicate and/or missing
           <td>
 If your application uses ADO.NET, you can install a combination of New Relic agents that will generate an ADO dashboard. Here’s what you need to do:
```

Comment:
If your application uses ADO.NET, you can install a combination of New Relic agents that will generate an ADO .net dashboard. Here’s what you need to do: